pitch.md

```diff
@@ -23,38 +51,192 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+The user will be able to define different rules that specify the coordinate where a 
+line starts, the direction it travels towards, and the distance it travels. Optional 
+things that the user can define is the speed of the line, the weight of the line, the
+color of the line, and whether or not the line will disappear as it travels (otherwise 
+default values will be set). Here is an example of the code I am envisioning:
+
+```
+// Creates two small spirals at the same time
+Start (50, 50) 
+Right: 10
+Down: 10
+Left: 15
+Up: 15
+Right: 20
+Down: 20
+Left: 25
+Up: 25
+
+Start (100, 50)
+Right: 10
+Down: 10
+Left: 15
+Up: 15
+Right: 20
+Down: 20
+Left: 25
+Up: 25
+```
+
+But I also want the user to be able to store and reuse a set of rules, so the code could be rewritten as follows:
+
+```
+// Creates two small spirals
```

Comment:
We like this one better because it's less verbose and reusable. It reminds me of ContextFree rules